### DISCUSSION TITLE
Hooks - Documented usage

### DISCUSSION THREAD

**Discussion by Rexeh** [marcelotduarte/cx_Freeze#1281] [Q&A] [answered]
Hi!

Thanks for the tool, it works great and not had any issues with virus scanners.

Regarding PyQT5 hooks, can you elaborate out the exact use of them? I'm currently leveraging the tool to build a PyQT5 application, with limited PyQT5 usage (Widgets/GUI/Core), but seeing the vast QT library being auto imported, bringing my package size up 50%.

Would the hooks resolve this situation? And if so could I get an example of their use in context of using the setup function / executable class.

**Comment by marcelotduarte** [comment, marked as answer]
Hi!
We have a sample for [pyqt5](https://github.com/marcelotduarte/cx_Freeze/tree/main/cx_Freeze/samples/pyqt5)
Regards of the size of the app, there is an open issue that I'll solve before releasing 6.9

**Reply by Rexeh** [reply]
Ace thanks for the quick reply
